### PR TITLE
docs: use new example configs to simplify docs

### PR DESCRIPTION
**Pull request by CharlieC3**
## Description

Updates the testnet and mainnet miner pages to make better use of the new example configs which will become available once this PR is merged: https://github.com/blockstack/stacks-blockchain/pull/2652

## Checklist

- [x] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] New links to files and images were verified: Can verify after above PR is merged.
- [x] For fixes/refactors: all existing references were identified and replaced
- [x] [Style guide](https://developers.google.com/style) was reviewed and applied
- [x] Clear code samples were provided
- [x] People were tagged for review

### CODE REVIEW

File: src/pages/references/stacks-node-configuration.md [hunk 1]
```diff
@@ -340,6 +340,36 @@ Example:
...
+#### read_only_call_limit_runtime
+
+Total execution cost limit for executing read-only smart-contracts.
+
+Example:
+
+```toml
+read_only_call_limit_runtime = 1000000000
+```
+
```
Comment (CharlieC3) [inline src/pages/references/stacks-node-configuration.md:372]: @lgalabru or @reedrosenbluth  Do these descriptions look correct to you? Also, what units is `read_only_call_limit_runtime` in?

File: src/pages/start-mining/mainnet.md [hunk 2]
```diff
@@ -92,43 +92,34 @@ bitcoin-cli -rpcport=8332 -rpcuser=your-user -rpcpassword=your-password importad
...
+...
 ```
 
 Now, grab your `privateKey` from earlier, when you ran the `make_keychain` command. Replace the `seed` field with your private key. Save and close this configuration file.
 
 To run your miner, run this in the command line:
 
 ```bash
-stacks-node start --config=./mainnet-miner-conf.toml
+stacks-node start --config=./testnet/stacks-node/conf/mainnet-miner-conf.toml
```
Comment (pgray-hiro) [inline src/pages/start-mining/mainnet.md:122]: Is the `testnet` string in this directory listing correct?  I see it's all through the mainnet mining doc so it makes me think it's not a mistake, but is it potentially confusing for someone who is intending to run a miner on the mainnet?

File: src/pages/start-mining/mainnet.md [hunk 3]
```diff
@@ -211,37 +202,28 @@ bitcoin-cli -rpcport=8332 -rpcuser=your-user -rpcpassword=your-password importad
 
 Once imported, we need to get some BTC to that address. You should be able to transfer BTC to this address using a crytpocurrency exchange such as [Coinbase](https://www.coinbase.com), [Binance](https://www.binance.com), or [Kraken](https://www.kraken.com).
 
-### Create configuration file
+### Update configuration file
 
-Now, we need to configure our node to use this Bitcoin keychain. In the **folder where your binary is extracted**, create a new file called `mainnet-miner-conf.toml`.
+Now, we need to configure our node to use this Bitcoin keychain. In the `stacks-blockchain` folder, modify the file at [`testnet/stacks-node/conf/mainnet-miner-conf.toml`](https://github.com/blockstack/stacks-blockchain/blob/master/testnet/stacks-node/conf/mainnet-miner-conf.toml).
```
Comment (pgray-hiro) [inline src/pages/start-mining/mainnet.md:207]: NAR: It's crazy that this content is repeated on this page, we need to do something about that in a future update.

**Review by pgray-hiro — APPROVED** [review]
A couple non-blocking comments, otherwise LGTM

File: src/pages/references/stacks-node-configuration.md [hunk 4]
```diff
@@ -340,6 +340,36 @@ Example:
...
+Example:
+
+```toml
+read_only_call_limit_read_count = 30
+```
+
+#### read_only_call_limit_runtime
+
+Total execution cost limit for executing read-only smart-contracts.
```
Comment (reedrosenbluth) [inline src/pages/references/stacks-node-configuration.md:365]: I'd reword this as such:

> [Runtime cost](https://github.com/stacksgov/sips/blob/2d3fd9bf8da7a04f588d90ff6252173d7609d7bf/sips/sip-006/sip-006-runtime-cost-assessment.md#introduction) limit for an individual read-only function call.

File: src/pages/references/stacks-node-configuration.md (hunk 1 above)
Comment (reedrosenbluth) [reply src/pages/references/stacks-node-configuration.md:372]: Left some comments above.

Runtime cost is a unitless metric. It loosely represents CPU cycles and exists as a basis for comparison. Learn more here: https://github.com/stacksgov/sips/blob/2d3fd9bf8da7a04f588d90ff6252173d7609d7bf/sips/sip-006/sip-006-runtime-cost-assessment.md#introduction

File: src/pages/start-mining/mainnet.md (hunk 2 above)
Comment (CharlieC3) [reply src/pages/start-mining/mainnet.md:122]: Yeah I think the Splat team knows the folder structure needs to be reworked, but right now this is correct. The [source code for the `stacks-node` binary is in this folder](https://github.com/blockstack/stacks-blockchain/tree/master/testnet), as well as the [example config files](https://github.com/blockstack/stacks-blockchain/tree/master/testnet/stacks-node/conf).

**Comment by CharlieC3** [comment]
Will merge after the dependent PR is merged.
